intellij-plugin-verifier/verifier-core/build.gradle.kts

```diff
@@ -0,0 +1,8 @@
+repositories {
```

Comment:
Will it be got from the root gradle file?

Reply:
Yes, this is inherited from `projects`. I will remove that.